src/docker/processor/config-template.yaml

```diff
@@ -0,0 +1,10 @@
+health_check_port: 8084
+server_config:
+  processor_name: econia_processor
+  postgres_connection_string: postgres://econia:econia@postgres:5432/econia
+  indexer_grpc_data_service_address: <MODIFY_ME>
```

Comment:
This should probably be something unique to each field so that it can be replaced by a build script.

Reply:
@elliottdehn addressed in 86251552